code/game/machinery/syndicatebeacon.dm

```diff
@@ -102,119 +102,163 @@
 		selfdestructing = 1
 		spawn() explosion(src.loc, 1, rand(1,3), rand(3,8), 10)
 
-
-
-#define SCREWED 32
-
-/obj/machinery/singularity_beacon //not the best place for it but it's a hack job anyway -- Urist
-	name = "ominous beacon"
-	desc = "This looks suspicious..."
+//Not the best place for it but it's a hack job anyway -- Urist
+/obj/machinery/singularity_beacon
+	name = "singularity beacon"
+	desc = "A suspicious-looking beacon. It looks like one of those snazzy state-of-the-art bluespace devices."
 	icon = 'icons/obj/singularity.dmi'
 	icon_state = "beacon"
-
 	anchored = 0
 	density = 1
-	layer = MOB_LAYER - 0.1 //so people can't hide it and it's REALLY OBVIOUS
-	stat = 0
+	machine_flags = WRENCHMOVE | FIXED2WORK
+
+	light_color = LIGHT_COLOR_RED
+	light_range_on = 2
+	light_power_on = 2
 
-	var/active = 0 //It doesn't use up power, so use_power wouldn't really suit it
+	var/obj/item/weapon/cell/cell
+	var/power_load = 1000 //A bit ugly. How much power this machine needs per tick. Equivalent to one minute on 30k W battery, two second ticks

```

Comment:
The singularity beacon needs live grid wiring and should never, at any point, be running off its room's APC